Doc/library/subprocess.rst

```diff
@@ -850,7 +866,64 @@ The :mod:`subprocess` module exposes the following constants.
    on the subprocess.
 
    This flag is ignored if :data:`CREATE_NEW_CONSOLE` is specified.
+   
+.. data:: ABOVE_NORMAL_PRIORITY_CLASS
+
+   A :class:`Popen` ``creationflags`` parameter to specify that a new process
+   will have an above average priority.
```

Comment:
You have to add ".. versionadded:: 3.7" here: same for each added constant below.